src/components/Button/index.js

```diff
@@ -67,13 +67,16 @@ export const Button = ({
         align,
         loading,
       )}
-      startIcon={icon ? <Icon name={icon} className={size} /> : null}
+      startIcon={
+        loading ? (
+          <Icon.Loading strokeWidth={1} size={18} color={loaderColor} trackColor="white" />
```

Comment:
This will change the size of the overall button while loading. This might look like a glitch.. Might break the layout if not handled properly.